### PR TITLE
bump lazy_static to 1

### PR DESCRIPTION
**Pull request by ignatenkobrain**
Signed-off-by: Igor Gnatenko <i.gnatenko.brain@gmail.com>

### CODE REVIEW

File: Cargo.lock [hunk 1]
```diff
@@ -254,6 +260,11 @@ dependencies = [
  "winapi 0.2.8 (registry+https://github.com/rust-lang/crates.io-index)",
 ]
 
+[[package]]
```
Comment (tmccombs) [inline Cargo.lock]: Why does this ad d the shell-escape package?

Comment (ignatenkobrain) [reply Cargo.lock]: to be honest, I don't know. I just run `cargo update -p lazy_static`

**Comment by sharkdp** [comment]
Is there any specific reason why you want to update this dependency?

**Comment by ignatenkobrain** [comment]
There are several:

1) It is first stable version (1.0), so no more unexpected breakages after version bump
2) I package fd for Fedora and we try to not keep old versions of crates

**Comment by sharkdp** [comment]
Okay, thanks.

Could you please rebase this on top of master. This should hopefully make Travis pass.

**Comment by sharkdp** [comment]
I've included this change in a recent update to Cargo.toml on master, so this is not needed anymore. Thanks!

**Pull request closed without merging**